restli-int-test-api/src/main/idl/com.linkedin.restli.examples.greetings.client.associations.restspec.json

```diff
@@ -83,6 +83,77 @@
             "path" : "/associations/{associationsId}/associationsSub/{subKey}"
           }
         }
+      }, {
+        "name" : "associationsassociations",
```

Comment:
nit: the first sub resource is with camel case, I think it might be better read if we make it same pattern?
`associationsassociations` maybe to `associationsAnotherSub` or whatever makes consistent.

Reply:
change to associationsAssociations